### PR TITLE
Modified Barrel TOF geometry to add dead spaces between sensors and between support ring structure

### PR DESCRIPTION
**Pull request by ssedd1123**
### Briefly, what does this PR introduce?

The BTOF active volume is modified to be more realistic. Now there are dead spaces between each sensors on a stave and extra dead space is reserved in the center of the BTOF to make space for the central supporting ring structure. 

Variables relevant to the dead spaces, sensor sizes and the width of the ring are adjustable from tof_barrel.xml. 

image

### What kind of change does this PR introduce?
- [ ] Bug fix (issue #__)
- [ ] New feature (issue #__)
- [ ] Documentation update
- [x ] Other: BTOF geometry is modified to accommodate individual sensors. 

### Please check if this PR fulfills the following:
- [ ] Tests for the changes have been added
- [ ] Documentation has been added / updated
- [ ] Changes have been communicated to collaborators

### CODE REVIEW

**Review by DevilsZ — APPROVED** [review]

**Comment by DevilsZ** [comment]
Dear @veprbl 

I'm not sure you are the right person to ask, but I just want to ask why this MR is not approved?
What should we fix? Or to whom should we ask?

File: src/BarrelTOFTracker_geo.cpp [hunk 1]
```diff
@@ -0,0 +1,399 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2022 Whitney Armstrong
```
Comment (veprbl) [inline src/BarrelTOFTracker_geo.cpp:2]:
```suggestion
// Copyright (C) 2022 - 2024, Whitney Armstrong, Chun Yuen Tsang
```

Comment (veprbl) [reply src/BarrelTOFTracker_geo.cpp:2]: @ssedd1123 This is to acknowledge that your contribution is licensed under LGPL3+.

**Comment by veprbl** [comment]
@ssedd1123
The following needs to be done.
1. We need to add your account to the https://github.com/eic organization, for that please have your real name and affiliation on your account. We can send you invite after that.
2. Please push your contribution as a branch to the shared https://github.com/eic/epic/ repository (e.g. `git checkout -b pr/tof_dead_spaces`, `git remote set-url origin git@github.com:eic/epic.git`, `git push -u origin pr/tof_dead_spaces`).
3. Re-submit a PR using that branch. This way you will have permission to run benchmarks and your contribution will be mergeable.

**Comment by ssedd1123** [comment]
> @ssedd1123 The following needs to be done.
> 
> 1. We need to add your account to the https://github.com/eic organization, for that please have your real name and affiliation on your account. We can send you invite after that.
> 2. Please push your contribution as a branch to the shared https://github.com/eic/epic/ repository (e.g. `git checkout -b pr/tof_dead_spaces`, `git remote set-url origin git@github.com:eic/epic.git`, `git push -u origin pr/tof_dead_spaces`).
> 3. Re-submit a PR using that branch. This way you will have permission to run benchmarks and your contribution will be mergeable.

I updated my profile with my name and affiliation. Please send me the invitation. Thank you and I greatly appreciate your help.

**Comment by DevilsZ** [comment]
Dear @veprbl ,

Could you also send me invitation to the eic git repo as I'm a ToF software coordinator?
I also added real name and affiliation to my github account. Thanks in advance.

**Comment by veprbl** [comment]
#787 has been posted, let's close this.

**Comment by freee458** [comment]
Dear @veprbl ,

Could you send me invitation to the eic GitHub.
I am working on ToF Endcap simulations. I have added my name and name of the Institute I am affiliated to. Thank you.

**Pull request closed without merging**